app/src/main/java/org/simple/clinic/contactpatient/ContactPatientEffectHandler.kt

```diff
@@ -19,6 +21,7 @@ class ContactPatientEffectHandler @AssistedInject constructor(
     private val appointmentRepository: AppointmentRepository,
     private val clock: UserClock,
     private val schedulers: SchedulersProvider,
+    private val currentFacilityStream: Observable<Facility>,
```

Comment:
We don't have to listen to facility changes here. So we can just inject `dagger.Lazy<Facility>` here.